src/components/maincontroller.js

```diff
@@ -204,9 +194,7 @@ window.castReceiverContext.addEventListener(
                 JSON.stringify(event.data)
         );
 
-        if ($scope.userId != null) {
-            reportEvent('volumechange', true);
-        }
+        reportEvent('volumechange', true);
```

Comment:
I guess we could do `if (JellyfinApi.userId)` - but I removed so many "do we have a connection" checks. I'm assuming someone would connect before these calls would happen.